test/scripts/1/s2.lua

```diff
@@ -1,13 +1,18 @@
 -- @interval 5s
 
+local alert = require('alert')
```

Comment:
I am working on massive refactoring regarding integration tests. I think we’ll add integration tests later, ok?

Reply:
Ok reverted my changes.